steal.lic

```diff
@@ -335,36 +372,36 @@ class Steal
               'you decide to wait a while before trying again',
               'Roundtime')
     when /this trivial theft/
-      update_target(target, 'trivial')
+      update_target(target, 'trivial') if count == 1
```

Comment:
Why did you add the conditional here?

Reply:
These were being called multiple times per theft.  Usually that's harmless, but for a new item it was making N copies if you stole N times.